torch/optim/nadamw.pyi

```diff
@@ -0,0 +1,14 @@
+from typing import Tuple
+
+from .optimizer import _params_t, Optimizer
+
+class NAdamW(Optimizer):
```

Comment:
There is no reason for it have it's own typing stub, just use the type annotations in the .py file